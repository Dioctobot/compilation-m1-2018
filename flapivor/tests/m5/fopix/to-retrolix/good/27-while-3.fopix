val main =
    val r = allocate_block (1);
    r[0] := 0;
    while (r[0] < 5) do
      val s = allocate_block (1);
      s[0] := 0;
      while (s[0] < 5) do
        s[0] := s[0] + 1;
        print_int (r[0] + s[0])
      end;
      r[0] := r[0] + 1
    end