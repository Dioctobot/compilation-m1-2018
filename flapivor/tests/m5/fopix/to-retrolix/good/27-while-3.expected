1234523456345674567856789